Notification Service/NotificationService.swift

```diff
@@ -15,19 +15,41 @@ import Combine
 class NotificationService: UNNotificationServiceExtension {
 
     private var cancellables = Set<AnyCancellable>()
+    var contentHandler: ((UNNotificationContent) -> Void)?
```

Comment:
The idea here is that we always have something to fall back on, and always try to add the intent, despite failing at any stage.